{
  "contest": 126,
  "handle": "unforgiven",
  "address": "0x6e6c53f81e310d53387dc2166ee9a632cf768fa1",
  "risk": "3",
  "title": "In VE3DRewardPool it's not possible to remove rewardTokens and if there were one bad rewardToken (by mistake or token related contract chaning) then no one can withdraw any rewards from VE3DRewardPool",
  "issueId": 179,
  "issueUrl": "https://github.com/code-423n4/2022-05-vetoken-findings/issues/179"
}